Add exposing metrics via HTTP endpoint (#901)
memgraph/docs: docs/reference-guide/enterprise-features.md

- authentication and authorization (username/password verification and user to
  role mapping)

## Exposing system metrics

> For [exposing system metrics](/reference-guide/exposing-system-metrics.md), we have built
> an HTTP endpoint where system metrics can be fetched via a GET request. The response contains
> information about bolt messages, sessions, transactions, streams, triggers, indexes, snapshots, 
> query latency, and many more.

Reviewer: ```suggestion
HTTP endpoint allows you to fetch [system metrics](/reference-guide/exposing-system-metrics.md) using a GET request. The response contains information about bolt messages, sessions, transactions, streams, triggers, indexes, snapshots, 
query latency, and many more.
```

User oriented vs. "we built"
